config_sample.toml

```
@@ -0,0 +1,26 @@
+baudrate = 1000000
+
+# # --- Network / addressing ---
+# dotbot_address = "FFFFFFFFFFFFFFFF"
+# gateway_address = "0000000000000000"
+network_id = "A000"
+
+# # --- Controller HTTP interface ---
+# controller_http_protocol = "http"
+# controller_http_hostname = "localhost"
+# controller_http_port = 8000
+
+# --- MQTT configuration ---
+mqtt_host = "argus.paris.inria.fr"
+mqtt_port = 8883
+mqtt_use_tls = true
```

```suggestion
mqtt_host = "localhost"
mqtt_port = 1883
mqtt_use_tls = false
```

Otherwise this is too much specific to our setup. This way the cloud adapter can also be used with a local MQTT broker, even when offline.